src/libs/SearchUIUtils.ts

```diff
@@ -556,6 +558,10 @@ function getSuggestedSearchesVisibility(
         const isExporter = policy.exporter === currentUserEmail;
         const isApprover = policy.approver === currentUserEmail;
         const isApprovalEnabled = policy.approvalMode ? policy.approvalMode !== CONST.POLICY.APPROVAL_MODE.OPTIONAL : false;
+
+        const hasExportError = (Object.keys(policy?.connections ?? {}) as ConnectionName[]).some((connection) => {
+            return !!hasSynchronizationErrorMessage(policy, connection, false);
+        });
```

Comment:
Nit: There's a check to early return, so policy is non-null. The function `hasSynchronizationErrorMessage` has explicit boolean return type, so I think we can omit `!!`.
```suggestion
        const hasExportError = (Object.keys(policy.connections ?? {}) as ConnectionName[]).some((connection) => {
            return hasSynchronizationErrorMessage(policy, connection, false);
        });
```